Doppler.Files.Api/docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+﻿version: "3.9"  # optional since v1.27.0
```

Comment:
In my opinion, this file is not necessary. Already exist the docker-compose.yml at the solution level.
For what could be used?